docs/userGuide/plugins/algolia.mbdf

```diff
@@ -0,0 +1,34 @@
+#### `algolia`: Enabling Algolia Docsearch
+
+This plugin allows you to use [Algolia Docsearch](https://community.algolia.com/docsearch/) for your site.
+
+To enable it, add `algolia` to your site's plugins, and provide your docsearch API key and index name in the plugins context. 
+
+```json
+site.json
+
+{
+  ...
+  "plugins": [
+    "algolia"
+  ],
+  "pluginsContext": {
+    "algolia": {
+      "apiKey": ALGOLIA_API_KEY,
+      "indexName": ALGOLIA_INDEX_NAME
```

Comment:
We can make it clearer to the user that they are supposed to replace these values? Eg. `INSERT_AGOLIA_API_KEY`, `INSERT_AGOLIA_INDEX_NAME`.